src/marketing/event-restrict-sales.md

```diff
@@ -3,6 +3,6 @@ conditions: Default.EE-B2B
 title: Sales Restrictions
 ---
 
-By default, products that appear in upcoming or closed events are not available for general sale, and the Add to Cart button does not appear on the product list or product page.
+By default, products that appear in upcoming or closed events are not available for general sale and the _Add to Cart_ button does not appear on the product list or product page.
 
-To restore the Add to Cart button for a closed event, the event must be deleted. However, if a product is associated with another category that has no selling restrictions, the Add to Cart button does appear on the product page. Similarly, the ticker block does not appear on the product page if the product is associated with another category that has no selling restrictions.
+To restore the _Add to Cart_ button for a closed event, the event must be deleted. However, if a product is associated with another category that has no selling restrictions, the Add to Cart button is available on the product page. Similarly, the ticker block does not appear on the product page if the product is associated with another category that has no selling restrictions.
```

Comment:
for must be deleted, do we need to add information on how to do that?

Reply:
We don't have a "Delete" procedure anywhere. Maybe we should add to the "Updating Events" topic. Let's open a GH issue for this and then we can add the link from this page.